pt-BR/mail/chrome/messenger/filter.properties

```diff
@@ -53,20 +53,20 @@ filterAction17=aplicada tag
 #entities below were automatically added by l10nmerge
 
 
-bodyFilterWarning=Creating a Body filter will automatically mark the Inbox for Offline use in Folder Properties. For this filter to operate on other folders, you must manually set those folders for Offline use in Folder Properties.
-bodyFilterWarningCheck=Don't show this warning again
-systemSound=System Sound
-soundFilePickerTitle=Choose Sound
-installScriptCheck=Yes, please install it
-noScriptSelected=Select an AppleScript
-bodyFilters=Body Filters
-filterAction18=removed topic
-scriptFilePickerTitle=Choose Script
-filterAction20=mark as pending
-installScripLabel=You have selected %S. Do you want to install it in your personal Scripts folder?
-scriptFiles=AppleScript Files
-filterAction19=played sound
-installScript=Install Script in your personal Scripts folder?
-filterAction21=launched script
-filterAction22=mark as
-filterAction23=quick post
+bodyFilterWarning=Criar um filtro no Corpo da mensagem marcará automaticamente a Caixa de Entrada para uso offline nas propriedades da pasta. Para que este filtre funcione em outras pastas, você deve configurá-las manualmente para uso offline nas propriedades da pasta.
```

Comment:
CORRECTIONS NEEDED it should be bodyFilterWarning=Criar um filtro no Corpo da mensagem marcará automaticamente a Caixa de Entrada para uso offline nas propriedades da pasta. Para que este filtro funcione em outras pastas, você deve configurá-las manualmente para uso offline nas propriedades da pasta.